sqlglot/dialects/dialect.py

```diff
@@ -181,11 +181,7 @@ def generator(self, **opts):
 
 def rename_func(name):
     def _rename(self, expression):
-        args = (
-            expression.expressions
-            if isinstance(expression, exp.Func) and expression.is_var_len_args
-            else expression.args.values()
-        )
+        args = list(flatten(expression.args.values()))
```

Comment:
i don't think you need the list here right

Reply:
right thanks